dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DependencyTreeFormatter.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.dashboard;
+
+import com.google.cloud.tools.opensource.dependencies.DependencyGraph;
+import com.google.cloud.tools.opensource.dependencies.DependencyGraphBuilder;
+import com.google.cloud.tools.opensource.dependencies.DependencyPath;
+import java.io.OutputStream;
+import java.io.PrintStream;
+import java.util.List;
+import org.apache.commons.lang3.StringUtils;
+import org.eclipse.aether.artifact.DefaultArtifact;
+import org.eclipse.aether.collection.DependencyCollectionException;
+import org.eclipse.aether.resolution.DependencyResolutionException;
+
+public class DependencyTreeFormatter {
+
+  public static void main(String[] args) {
+    if (args.length < 1) {
+      System.err.println("Artifact coordinate not provided. E.g., 'io.grpc:grpc-auth:1.15.0'");
+      System.exit(1);
+    }
+    String coord = args[0];
```

Comment:
perhaps loop over args to handle multiple coordinates

Reply:
Google Java style is to not abbreviate variable names; thus coordinates, not coord